tests/snuba/api/endpoints/test_organization_group_index.py

```diff
@@ -739,21 +742,8 @@ def test_inbox_fields(self):
                 data={"timestamp": iso_format(before_now(seconds=500)), "fingerprint": ["group-1"]},
                 project_id=self.project.id,
             )
-            self.store_event(
```

Comment:
just curious cuz i don't touch the backend as often, but were these not needed?

Reply:
Yeah they were totally irrelevant to what was actually being tested and seemed fine to be removed (...aka I don't know why I put them there in the first place 😄 )